std_msgs/String order
std_msgs/Float64 speed
std_msgs/Float64 acceleration
std_msgs/Float64 distance
std_msgs/Float64 radius
std_msgs/String direction
std_msgs/Float64 target_speed
std_msgs/Float64 omega
std_msgs/Float64 angle
std_msgs/Float64 time
---

std_msgs/Bool result
